jobserver/api/jobs.py

```diff
@@ -200,10 +211,22 @@ def handle_alerts_and_notifications(request, job_request, job, github_api):
             user_id=job_request.created_by_id,
         )
 
+
+def handle_job_request_notifications(job_request, github_api):
```

Comment:
Should this section from `handle_job_notifications` be moved down here?

We'll have to tell people that it's changed, but I think they'd probably thank us?

```
if job_request.will_notify:
        send_finished_notification(
            job_request.created_by.notifications_email,
            job,
        )
        logger.info(
            "Notified requesting user of completed job",
            user_id=job_request.created_by_id,
        )
```

Reply:
Done in #2848